HIRS_Utils/src/main/java/hirs/validation/SupplyChainCredentialValidator.java

```diff
@@ -678,6 +688,26 @@ private static boolean requiredPlatformCredentialFieldIsNonEmptyAndMatches(
             return false;
         }
 
+        return platformCredentialFieldMatches(platformCredentialFieldName,
+                platformCredentialFieldValue, otherValue);
+    }
+
+    private static boolean notRequiredPlatformCredentialFieldMatchesNotNullAndMatches(
```

Comment:
I would say that this method should get Javadoc similar to the one it mirrors above for consistency. Also, I would consider renaming this to something that's a bit easier to understand `notRequired` could be `optional`.

Additionally, the name says it matches not null, but it looks to match explicitly on null, so perhaps the name should be `optionalPlatformCredentialFieldNullOrMatches`.